test/framework/lib.py

```diff
@@ -54,7 +54,8 @@ def setUp(self):
         super(EasyBuildLibTest, self).setUp()
 
         # make sure BuildOptions instance is re-created
-        del BuildOptions._instances[BuildOptions]
+        if BuildOptions in BuildOptions._instances:
+            del BuildOptions._instances[BuildOptions]
```

Comment:
Shouldn't the "del BuildOptions..." in configure further down be protected too just in case?

Reply:
sure, why not, fixed in 852cdd3